src/javascript/crypto/e2e/extension/constants.js

```diff
@@ -23,6 +23,7 @@ goog.provide('e2e.ext.constants.Actions');
 goog.provide('e2e.ext.constants.BackupCode');
```

Comment:
Why is that file needed? These namespaces are not used elsewhere (+ it's in the different, `ext` namespace).